src/css/App.scss

```diff
@@ -0,0 +1,424 @@
+    font-family: 'Roboto', sans-serif;
+    width: fit-content;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 18px;
+    align-self: flex-end;
+
+    line-height: 21px;
+    transition: 0.1s;
+   }
+  
+   button:hover {
+    background: #C5E9FF;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+
+    transition: 0.1s;
+    color: #fff;
+   }
+}
+
+.modalWrap{
+  background: rgba(196, 196, 196, 0.829);
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  width: 100vw;
+  opacity: 0;
+  pointer-events: none;
+  transition: 0.2s;
+}
+.modalWrap.active {
+  transition: 0.2s;
+  opacity: 1;
+  pointer-events: all;
+}
+.modal{
+  background-color: #fff;
+  transform: scale(0.5);
+  overflow: hidden;
+  transition: 0.2s;
+  width: 600px;
+  border-radius: 4px;
+  height: auto;
+}
+.modal.active{
+  transition: 0.2s;
+  transform: scale(1);
+
+}
+.svgWrap{ 
+  img {
+    transition: 0.1s;
+  }
+  img:hover {
+    transform: scale(1.1);
+    transition: 0.1s;
+  }
+
+}
+.modalEditBody {
+  display: flex;
+  flex-direction: column;
+  padding: 40px;
+  select {
+    background: #FFFFFF;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    box-sizing: border-box;
+    border-radius: 4px;
+    padding: 10px;
+  }
+  textarea {
+    resize: none;
+    padding: 20px;
+    overflow-y: scroll;
+  }
+  label {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 18px;
+    line-height: 21px;
+    margin-bottom: 10px;
+    margin-top: 20px ;
+  }
+  input {
+    background: #FFFFFF;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    box-sizing: border-box;
+    border-radius: 4px;
+    padding: 10px;
+  }
+}
+.modalBody {
+  
+  p {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 24px;
+    line-height: 28px;
+    margin: 30px 45px;
+    min-height: 80px;
+
+  }
+}
+.modalHeader {
+  display: flex;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 20px 30px;
+  p {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 36px;
+    line-height: 42px;
+    margin: 0;
+  }
+}
+.modalFooter {
+  display: flex;
+  box-shadow: 0px 0px 4px 4px rgba(0, 0, 0, 0.25);
+  
+  justify-content: flex-end;
+
+  padding: 20px 30px;
+  button {  
+    transition: 0.1s;
+    margin-left: 12px;
+    padding: 10px 45px;
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 18px;
+    line-height: 21px;
+    background-color: #fff;
+    border: 2px solid rgba(0, 0, 0, 0.2);
+    box-sizing: border-box;
+    border-radius: 4px;
+  }
+  button:hover {
+    transition: 0.1s;
+    border: 2px solid rgba(197, 233, 255, 0.72);
+    background: rgba(197, 233, 255, 0.72);
+  }
+
+}
+
+@keyframes errPopUp {
+  from {
+    transform: scale(1.03);
+    opacity: 0;
+    
+  }
+
+  to {
+    transform: scale(1);
+    opacity: 1;
+  }
+}
```

Comment:
please check all the styles for the design. And nowhere should be the exact values